packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.vitest.tsx

```diff
@@ -59,47 +54,21 @@ const renderComponent = (
   return render(getComponent(props), { wrapper: createWrapper(synapseContext) })
 }
 
-function mockGetEntityWiki(markdown: string) {
-  server.use(
-    rest.get(
-      `${getEndpoint(
-        BackendDestinationEnum.REPO_ENDPOINT,
-      )}/repo/v1/entity/:id/wiki/:wikiId`,
-      async (req, res, ctx) => {
-        const response: WikiPage = {
-          ...mockEntityWikiPage,
-          markdown,
-        }
-        return res(ctx.status(200), ctx.json(response))
-      },
-    ),
-  )
-}
+const spyOnMath = vi.spyOn(MarkdownSynapse.prototype, 'processMath')
 
-describe('MarkdownSynapse tests', () => {
-  beforeAll(() => server.listen())
-  afterEach(() => server.restoreHandlers())
-  afterAll(() => server.close())
-
-  const mockGetWikiAttachments = jest.spyOn(
-    SynapseClient,
-    'getWikiAttachmentsFromEntity',
-  )
+mockSynapseClient.getWikiAttachmentsFromEntity.mockResolvedValue({ list: [] })
 
+describe('MarkdownSynapse tests', () => {
```

Comment:
One of the reasons I picked this test is because (in a future PR) I want to dynamically import the MarkdownWidget component implementations to break a dependency cycle and enable code-splitting in an experiment I am working on. Vite lets you do it, but [Jest's support is experimental](https://jestjs.io/docs/next/ecmascript-modules) and is not trivial to configure. We need to move to Vitest anyways, so this is a good place to start.

Reply:
Like to hear more about the identified dependency cycle that is blocking code-splitting.  Nice work

Reply:
One example is
MarkdownSynapse -> MarkdownWidget -> QueryWrapperPlotNav -> ... -> MarkdownSynapse

I think there are a few places this can happen, such as in cards